VERSIONING.md

```diff
@@ -101,7 +101,9 @@ Major and Minor Breaking Changes
 
     * A breaking change in Customization API is understood as a
         **minor breaking change**.
-
+* **Build and toooling:**
+  * NodeJS Updates are considered a major breaking change. We will try to stick to
+    LTS versions.
```

Comment:
```suggestion

* **Build and Tooling:**

  * Node.js updates are considered a major breaking change. We will try to stick
    to LTS versions.

```

Adjusted to:

* [Title case](https://capitalizemytitle.com/).
* Fix typo.
* Adjust spacing (better readable in text-mode).
* Limit chars per line to 80 (recommended for reading at console)
* Adjusted name: NodeJS → Node.js (this seems to be the official name)